app/tests/integration/e2e_test_variables.json

```diff
@@ -24,5 +24,33 @@
                 "firefox_pref_branch": "default"
             }
         ]
+    },
+    "multi-pref-experiment": {
+        "action_name": "multi-preference-experiment",
+        "name": "E2E experimenter multipref-test",
+        "type": "channel",
+        "channels": "Nightly",
+        "min_version": "80.0",
+        "max_version": "100.0",
+        "userFacingName": "e2e testing name",
+        "userFacingDescription": "e2e testing description",
+        "branches": [
+            {
+                "branch_description": "multipref-branch-description",
+                "branch_name": "e2e-multiepref-branch",
+                "firefox_pref_branch": "default",
+                "firefox_pref_name": "e2e-testing",
+                "firefox_pref_type": "boolean",
+                "firefox_pref_value": "true"
+            },
+            {
+                "branch_description": "multipref-branch-description-2",
+                "branch_name": "e2e-multiepref-branch-2",
+                "firefox_pref_branch": "default",
+                "firefox_pref_name": "e2e-testing",
+                "firefox_pref_type": "boolean",
+                "firefox_pref_value": "false"
```

Comment:
Can you put 2-3 prefs in each branch?  I'd like to stress test that they all get in.

Reply:
We could but that may be out of scope. Maybe a multi-pref multipref e2e test?

Reply:
I could add that right after this

Reply:
I don't understand what you mean.  How is testing multiple prefs per branch out of scope for the multipref e2e test?  Isn't that exactly what this should be testing?

Reply:
Right now we are, but adding multiple prefs would be testing something else as well. This is the minimum case for a multipref experiment. Adding more prefs should be its own test imo

Reply:
I disagree, of the three cases you're describing:

1) Same pref for all branches (already tested)
2) Different prefs per branch with 1 pref per branch
3) Different prefs per branch with multiple prefs per branch


1 and 3 are the important cases, 2 is the edge case, so I think 1 and 3 are the tests we should have now before we start considering new test cases.  Although I didn't provide details in the bug, the specific thing I was looking for in this test is case 3.

Reply:
But 2 leads to 3. So why not start with 2? I was trying to create minimum test cases for each of these e2e tests, and then expand them to be specific later. But having a minimum test case tested is where we should start.

Reply:
Is 1 pref per branch not something that usually happens?

Reply:
Not with 1 unique pref per branch, no.  Either it's 1 pref, but it's the same pref for both branches (the single pref case), or it's multiple prefs and there's multiple copies of the same pref in each branch (case 3).  Case 2 doesn't generally happen.

And I get your logic about progressively increasing cases, but each test has a cost, in terms of development time, and in terms of run time, so I think it's prudent to be conservative with adding cases and right now we have a concrete list of the core cases I think are the minimum surface to ensure everything's behaving correctly, and those are cases 1 and 3.

Reply:
Alrighty, I will add 2 prefs ber branch. Is that more in line with what would work best?

Reply:
@jrbenny35 Yes exactly, thank you!